### PR TITLE
data: Mohiit70

### PR DESCRIPTION
**Pull request by Mohiit70**
## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Hi Eddie,

Thank you for taking the time to review my pull request. This is my first contribution to an open-source project, and I'm excited to be a part of this community.

Please let me know if there's anything that needs to be fixed or improved. I'm open to feedback and eager to learn.

Thank you again for your time and guidance.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5568"></a>

### CODE REVIEW

**Review by iamthecloverly — APPROVED** [review]

File: data/Mohiit70.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Mohit Bisht",
+  "type": "personal",
+  "bio": "Passionate about Learning, Creating, and Contributing to the World of Web Development and Cloud.",
+  "followMe": [
```
Comment (Ananya2001-an) [inline data/Mohiit70.json:5]: It has to be links key.
```suggestion
  "links": [
```

File: data/Mohiit70.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohit-bisht70/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Mohiit70",
+      "icon": "FaGitHub"
```
Comment (Ananya2001-an) [inline data/Mohiit70.json:19]: same here..
```suggestion
      "icon": "FaGithub"
```

File: data/Mohiit70.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name": "Follow me on hashnode",
+      "url": "https://mohitbisht70.hashnode.dev/",
+      "icon": "Fahashnode"
```
Comment (Ananya2001-an) [inline data/Mohiit70.json:24]: Not sure but I guess this should also be something like this.
```suggestion
      "icon": "FaHashnode"
```

**Review by Ananya2001-an — DISMISSED** [review]
Looks great! Thanks for making the changes 👍🏼

File: data/Mohiit70.json [hunk 4]
```diff
@@ -11,17 +11,17 @@
...
       "name": "Follow me on GitHub",
       "url": "https://github.com/Mohiit70",
-      "icon": "FaGitHub"
+      "icon": "FaGithub"
     },
     {
       "name": "Follow me on hashnode",
       "url": "https://mohitbisht70.hashnode.dev/",
-      "icon": "Fahashnode"
+      "icon": "FaHashnode"
```
Comment (khushalarora) [inline data/Mohiit70.json:24]: Hi @Mohiit70, thank you for contributing  :tada: 
Please replace this with  **' SiHashnode '** 
You can checkout [ICONS](https://linkfree.eddiehub.io/icons) to search icons.  :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]

**Review by khushalarora — APPROVED** [review]
Looks good, thanks  :tada:

**Pull request merged**